lib/client/api.go

```diff
@@ -326,7 +326,9 @@ func (tc *TeleportClient) SSH(command []string, runLocally bool) error {
 }
 
 // Join connects to the existing/active SSH session
-func (tc *TeleportClient) Join(sessionID session.ID) (err error) {
+func (tc *TeleportClient) Join(sid string) (err error) {
```

Comment:
The session ID is always provided _by a user via UI_ for `Join()` implementation. That's why it makes no sense to require `session.ID` type